cmd/podman/varlink.go

```diff
@@ -24,7 +24,7 @@ var (
 	varlinkFlags = []cli.Flag{
 		cli.IntFlag{
 			Name:  "timeout, t",
-			Usage: "time until the varlink session expires in milliseconds. default is 1 second; 0 means no timeout.",
+			Usage: "time until the varlink session expires in milliseconds.  Use 0 to disable the timeout.",
```

Comment:
I like your rewording for the zero value, but would like to see the default value (1 sec) that you get if you don't define something.  Would 'Use 0 to disable the timeout, unless specified, 1 second is the default.'  work for you?

Reply:
hmm doesn't seem to be honored by podman though

Reply:
no nevermind it was my mistake, it works.